hritam-mondal/TestTrek
# Add Authentication and Authorization Feature

# Add Authentication and Authorization Feature

## Description
This pull request introduces the authentication and authorization feature to the application.

### Changes Included
- Implemented user login and registration endpoints
- Integrated JWT authentication for secure user sessions
- Updated configuration to manage secrets and environment variables

### Screenshots
![image](https://github.com/user-attachments/assets/de3003bc-88c7-44f2-bf5b-1dc932dfdf8c)

## How to Test
Outline the steps to test the changes made in this pull request:
1. Clone the repository: `git clone https://github.com/hritam-mondal/TestTrek.git`
2. Checkout to the feature branch: `git checkout feature/auth`
3. Set up the environment and configuration variables as outlined in the README
4. Run the application: `dotnet run`
5. Verify that auth and user endpoints are working as expected
